docs.html

```diff
@@ -218,6 +218,75 @@ <h6 class="card-title">Macbook Air M2, 32GB</h6>
                     sandbox="allow-scripts allow-same-origin">
                 </iframe>
 
+                <h3 class="text-h3 sub-heading">Dismissable Card</h3>
+
+                <div class="card-output-container" id="card-dismiss">
+                    <div class="card-container">
+                        <img class="card-image" src="https://picsum.photos/id/2/300/200" alt="Item Image">
+                        <h6 class="card-title">Macbook Air M2, 32GB</h6>
+                        <p class="card-description">Apple M2 Chip with 8‑Core CPU and 7‑Core GPU
+                            256GB Storage</p>
+                        <div class="card-btn-container">
+                            <button class="card-btn card-btn-primary">Add to Cart</button>
+                            <button class="card-btn card-btn-secondary">Save For Later</button>
+                            <i class="card-btn-icon far fa-heart"></i>
+                            <i class="card-btn-icon fas fa-ellipsis-v"></i>
+                        </div>
+                        <div class="card-badge">-22%</div>
+                        <i class="card-dismiss-icon fas fa-times"></i>
+                    </div>
+
+                    <!--
```

Comment:
Can you remove this comment? and give this code on the UI . So usEr can use this.

Reply:
okay